albo.py

```diff
@@ -14,11 +15,42 @@ def escape_char (text: str, char_to_escape: List[str] = ['_', '*', '[', '`']) ->
 def send_telegram_message(text: str) -> None:
     params = {
         'chat_id': CHATID,
-        'text': text
+        'text': text,
+        'parse_mode': 'markdown'
     }
-    URL = "https://api.telegram.org/bot" + TOKEN + "/sendMessage?&parse_mode=markdown&" + urlencode(params, quote_via=quote_plus)
+    URL = "https://api.telegram.org/bot" + TOKEN + "/sendMessage?&" + urlencode(params, quote_via=quote_plus)
     r = requests.get(url = URL)
 
+def send_single_telegram_attachment(pdf_link: str) -> None:
+    params = {
+        'chat_id': CHATID,
+        'document': pdf_link,
+        'disable_notification': True
+    }
+    URL = "https://api.telegram.org/bot" + TOKEN + "/sendDocument?&" + urlencode(params, quote_via=quote_plus)
+    r = requests.get(url = URL)
+
+def send_multiple_telegram_attachments(pdf_links: List[str]) -> None:
+    params = {
+        'chat_id': CHATID,
+        'disable_notification': True,
+        'media': json.dumps([
+          { 
+            "type": "document",
+            "media": pdf_link
+          }
+        for pdf_link in pdf_links]),
+
+    }
+    URL = "https://api.telegram.org/bot" + TOKEN + "/sendMediaGroup?&" + urlencode(params, quote_via=quote_plus)
+    r = requests.get(url = URL)
+
+def send_telegram_attachments(pdf_links: List[str]) -> None:
+  if len(pdf_links) == 1:
+    send_single_telegram_attachment(pdf_links[0])
+  else:
+    send_multiple_telegram_attachments(pdf_links)
```

Comment:
use ternary operator and reduce teh redundanci of `send_single_telegram_attachment`
```suggestion
  attachment = pdf_links if not len(pdf_links) == 1 else pdf_links[0]
  send_single_telegram_attachment(attachment)
```

Reply:
This is not exactly correct, since I change also the method name to call, not just the argument passed, maybe I could try something like:
`send_single_telegram_attachment(pdf_links[0]) if len(pdf_links) == 1  else send_multiple_telegram_attachments(pdf_links) `

Reply:
that's ok